### PR TITLE
Add scripts to dump and load database to mysql server role.

### PR DESCRIPTION
**Pull request by nickygerritsen**
I'm always creating these scripts, so why not do it in Ansible.

### CODE REVIEW

File: icpc-wf/ansible/roles/mysql_server/tasks/main.yml [hunk 1]
```diff
@@ -25,3 +25,30 @@
...
+    state: directory
+    mode: 0755
+
+- name: create directory to store scripts
+  file:
+    path: /home/domjudge/bin
+    owner: domjudge
+    group: domjudge
+    state: directory
+    mode: 0755
```
Comment (vmcj) [inline icpc-wf/ansible/roles/mysql_server/tasks/main.yml:43]:
```suggestion
- name: create directory to store scripts & database dumps
  file:
    path: "/home/domjudge/{{ item }}"
    owner: domjudge
    group: domjudge
    state: directory
    mode: 0755
  loop:
    - db-dumps
    - scripts
```

File: icpc-wf/ansible/roles/mysql_server/tasks/main.yml [hunk 2]
```diff
@@ -25,3 +25,30 @@
...
+
+- name: copy database dump/load scripts
+  copy:
+    src: "{{ item }}"
+    dest: /home/domjudge/bin/{{ item }}
+    owner: domjudge
+    group: domjudge
+    mode: 0755
+  with_items:
```
Comment (vmcj) [inline icpc-wf/ansible/roles/mysql_server/tasks/main.yml:52]:
```suggestion
  loop:
```

File: icpc-wf/ansible/roles/mysql_server/tasks/main.yml [hunk 3]
```diff
@@ -25,3 +25,25 @@
...
+- name: create directory to store scripts & database dumps
+  file:
+    path: /home/domjudge/{{ item }}
+    owner: domjudge
+    group: domjudge
+    state: directory
+    mode: 0755
+  loop:
+    - db-dumps
+    - scripts
```
Comment (vmcj) [inline icpc-wf/ansible/roles/mysql_server/tasks/main.yml:38]: When is this actually used? In the next one you have bin, which on my VM is also not in my PATH.

Comment (nickygerritsen) [reply icpc-wf/ansible/roles/mysql_server/tasks/main.yml:38]: lol should be bin indeed. Wasn't it bin until I applied your change?
And my GCloud server and my Poli server both had it in bin. So we should maybe (later) figure out if we should add it there?

Comment (vmcj) [reply icpc-wf/ansible/roles/mysql_server/tasks/main.yml]: Yes. I think I introduced the error, makes reviewing so much more fun.

**Review by vmcj — APPROVED** [review]

**Review by eldering — APPROVED** [review]

**Pull request merged**